Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -175,7 +175,7 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
        .. parsed-literal::
 
-          curl -LO \ |SCM_WEB|\/Documentation/gettingstarted/kind-config.yaml
+          curl -LO \ |SCM_WEB|\/Documentation/advancedinstallation/kind-config.yaml
```

Comment:
Does `advancedinstallation` exist? At a glance it doesn't look like this is where the page ended up.

Reply:
Leftover from a previous naming scheme. I'll fix it, thanks for catching it!